.azure-pipelines.yml

```diff
@@ -203,6 +203,7 @@ jobs:
   - bash: |
       set -eux
       cargo test --features mock-time
+      cargo test --workspace
```

Comment:
don't we need to merge `cargo test --features mock-time` and `cargo test --workspace` ? I have the feeling we are running part of the tests two time here

And what tests are run by the `cargo test --features mock-time` ? given we run this command from the oxidation folder I would have guess everything within this folder is tested :/

Reply:
We should merge both together as `cargo test --workspace --features mock-time`
--features mock-time is not used for the moment, this enable the freeze_time feature

Reply:
done for merging both `cargo test`.

I wont give a list of which test were run before, but I can ensure that the tests in `platform_async` wasn't run